python/py-libtmux/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                py-libtmux
 version             0.8.2
-revision            0
+revision            1
```

Comment:
The `revision` does not need to be increased because the existing sub-ports are not changing.